cli/template/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 ink_core = { git = "https://github.com/paritytech/ink", package = "ink_core" }
 ink_model = { git = "https://github.com/paritytech/ink", package = "ink_model" }
 ink_lang = { git = "https://github.com/paritytech/ink", package = "ink_lang" }
-parity-codec = { version = "3.3", default-features = false, features = ["derive"] }
+parity-codec = { version = "4.1.1", default-features = false, features = ["derive"] }
```

Comment:
In general I would simply put `4.1` here. Also this has already been done in the latest set of changes to `master`.